src/helpers/fetchData.js

```diff
@@ -0,0 +1,30 @@
+const fetch = require('node-fetch');
+const { YOUTUBE_API_KEY, YOUTUBE_CHANNEL_ID } = require('../config');
+
+async function fetchLatestYoutubeVideos(options) {
+  const mrOpt = 'maxResults' in options ? `maxResults=${options.maxResults.toString()}` : `maxResults=10`;
+  const paOpt = 'publishedAfter' in options ? `publishedAfter=${options.publishedAfter.toString()}` : '';
```

Comment:
The .toString() here and on the previous line is redundant as you are coercing it already with the template literal.

Reply:
Didn´t know that. Amazing!